collectors/ebpf.plugin/README.md

```diff
@@ -219,9 +219,39 @@ The following options are available:
     range of IPs, or use CIDR values. The default behavior is to only collect data for private IP addresess, but this
     can be changed with the `ips` setting.
     
-By default, Netdata displays up to 500 dimensions on network viewer charts. If there are more possible dimensions, they
-will be bundled into the `other` dimension. You can increase the number of shown dimensions by changing the `maximum
-dimensions` setting.
+By default, Netdata `1.24.x` displays up to 49 sockets (dimensions) on network viewer charts. If there are more
+ dimensions, only the oldest ones will be shown individually and the rest will be bundled into the `other` dimension.
+ You can increase the number of shown dimensions by changing the `maximum dimensions` setting. 
+
+The dimensions for the traffic charts are created using the destination IPs of the sockets by default. This can be
+changed setting `resolve hostname ips = yes` and restarting Netdata, after this Netdata will create dimensions using
+the `hostnames` every time that is possible to resolve IPs to their hostnames.    
+
+#### Default values
+
+When the file `/etc/netdata/ebpf.conf` is missing, `ebpf.plugin` will start with the following default values
+
+```conf
+[network viewer]
+    maximum dimensions = 50
+    ports = *
+    hostnames = *
+    ips = !127.0.0.1/8 10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 fc00::/7
+```
+
+that will allow the collector to show connections for all ports and all hostnames on your network, on the other hand,
+ these settings also restricted the IP monitoring only for `private` IPs. 
+
+#### Network traffic chart
+
+Netdata monitors network connections established for hosts, which are shown according the package final destination.
+
+You can visualize the most common destinations for `outbound connections` and measure the bandwidth utilized with 
+connections for local services. All these information are grouped using:
+
+-   `bytes`: Bytes sent and received for a destination.
+-   `packets`: Packets sent per period of time.
+-   `retransmission`: Number of TCP packages retransmitted. This is available only for `outbound connections`. 
```

Comment:
> Number of TCP packages

It is `calls` on the charts. calls != packets. If this is `packets` then charts should be fixed, if `calls` - docs.

Reply:
> -   `packets`: Packets sent per period of time.

> per period of time.

It is per seconds in our case, if dimension algorithm is `incremental` then `netdata` calculates per second value.

I think `per period of time` is not needed at all and it is just `sent and received packets`

@joelhans

Reply:
I fixed the documentation.